### PR TITLE
Final order status2

### PR DESCRIPTION
**Pull request by paulgrever**

### CODE REVIEW

**Comment by paulgrever** [comment]
@spencern  will you check this out?

File: server/methods/orderDetail.js [hunk 1]
```diff
@@ -26,5 +26,43 @@ Meteor.methods({
         'advancedFulfillment.workflow.status': workflow[status]
       }
     });
+  },
+  'advancedFulfillment/orderCompleted': function (order, userId) {
```
Comment (spencern) [inline server/methods/orderDetail.js:30]: Let's add some tests for this method. @paulgrever

File: server/methods/orderDetail.js [hunk 2]
```diff
@@ -26,5 +26,43 @@ Meteor.methods({
...
+      $addToSet: {
+        'history': historyEvent,
+        'advancedFulfillment.workflow.workflow': 'orderInspected'
+      },
+      $set: {
+        'advancedFulfillment.workflow.status': 'orderCompleted'
+      }
+    });
+  },
+  'advancedFulfillment/orderIncomplete': function (order, userId) {
```
Comment (spencern) [inline server/methods/orderDetail.js:49]: Needs tests

File: server/methods/orderDetail.js (hunk 1 above)
Comment (spencern) [inline server/methods/orderDetail.js:30]: Let's also check permissions here before updating the order status.
e.g.

```
    if (!ReactionCore.hasPermission("advancedFulfillment")) {
      throw new Meteor.Error(403, "Access Denied");
    }
```

**Comment by spencern** [comment]
:+1: merging, can refactor later.

**Comment by paulgrever** [comment]
Cool. Started setting up base of app with all the packages and reading
package info

On Fri, Nov 6, 2015 at 12:08 PM, Spencer Norman notifications@github.com
wrote:

> Merged #99
> https://github.com/getoutfitted/reaction-advanced-fulfillment/pull/99.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/getoutfitted/reaction-advanced-fulfillment/pull/99#event-457263532
> .

## 

Paul Grever |Software Engineer |GetOutfitted  |m. 608-239-2471 |e.
paul@getoutfitted.com |w. getoutfitted.com
http://www.getoutfitted.com/?utm_source=Email-Signature-Rescue&utm_medium=Email-Signature&utm_campaign=Email-Signature-Rescue

**Pull request merged**